druid/src/window.rs

```diff
@@ -260,13 +260,18 @@ impl<T: Data> Window<T> {
             };
 
             self.root.lifecycle(&mut ctx, event, data, env);
+            self.post_event_processing(&mut base_state, queue, data, env, process_commands);
```

Comment:
we can move this out of the `if` branch if we move the `base_state` decl to above the block, this should solve our other little confusion?

Reply:
Yes! That will work.